shared/components/ember-output.jsx

```diff
@@ -0,0 +1,90 @@
+/* global React */
```

Comment:
I don't see any warnings in the tests or when serving.  I'll add it as a global anywhere to be safe.